upstairs/src/live_repair.rs

```diff
@@ -4334,9 +4261,11 @@ pub mod repair_test {
         // 2 Jobs were created
         assert_eq!(jobs.len(), 2);
```

Comment:
I'm surprised this didn't change?

Reply:
`create_and_enqueue_close_op` (which is only used in tests) now reserves job IDs for all four repair jobs, because it calls `get_repair_ids` (which gives you the set of four).

However, it only actually _enqueues_ the repair job (calling `create_and_enqueue_close_io`), and ignores the other three jobs.

I was debating how much to change this!  For example, we could write a `create_and_enqueue_repair_ops` function which enqueues all four repair jobs, then use it instead of `create_and_enqueue_{close,repair}_op` in tests.  This would more closely match our system behavior in `repair_extent`, although we can't match it exactly (because it actually waits for some jobs to complete before enqueueing others).  What do you think?

Reply:
Upon further discussion, I did switch to a `create_and_enqueue_repair_ops`, which enqueues all four repair jobs (with appropriate inter-job dependencies).  This makes the test suite match our actual code more closely.  See the changes in 4c514784a210bc6cb7f65fbcd908c7e814cba0a3